Request/index.ts

```diff
@@ -9,7 +9,7 @@ export interface Request {
 	readonly method: RequestMethod
 	readonly url: URL
 	readonly parameter: { readonly [key: string]: string }
-	readonly search: { readonly [key: string]: string }
```

Comment:
`search` is and should be mandatory. Use `Request.Like` when you are providing things in a lazy manner.